src/styles/block/cards.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &:nth-child(-n + 4) {
+    margin-right: 50px;
+    margin-bottom: 40px;
```

Comment:
probably wrong sizes of margins, it can be that u have failed npm tests also cuz of this, what do u think?